modules/cnv_master/1.0/src/R/merge_segs.R

```diff
@@ -22,25 +22,16 @@ message("Loading data from individual seg files ...")
 files <- snakemake@input[]
 
 # read individual files and keep file paths in the column filename
-data =
-tibble(filename = files$seg_file) %>% # create a data frame
-  # holding the file names
-  mutate(file_contents = map(filename,          # read files into
-                             ~ read_tsv(files$seg_file, col_types = cols())) # a new data column
-  ) %>%
-  unnest(cols = c(file_contents))
+data = lapply(files$seg_file, read_tsv, col_types = "ccddddd")
```

Comment:
very elegant, thanks!

Reply:
I only implemented this because map took FOREVER to run, and this is very fast. I don't understand why because I though map was essentially lapply.